rust/lance-core/src/io/stream.rs

```diff
@@ -22,6 +22,8 @@ use pin_project::pin_project;
 
 use crate::Result;
 
+pub type LanceRecordBatchStream = Pin<Box<dyn RecordBatchStream + Send + 'static>>;
```

Comment:
Should we just called it `PinedRecordBatchStream` or something. 

It is not clear why this one called `LanceRBS` but the one blow is not.

Reply:
Actually I think I can remove this.